opencti-platform/opencti-graphql/src/database/middleware.js

```diff
@@ -1933,7 +1945,7 @@ export const updateAttribute = async (context, user, id, type, inputs, opts = {}
         }
       } else {
         // Special access check for RELATION_GRANTED_TO meta
-        if (relType === RELATION_GRANTED_TO && !userHaveCapability(user, KNOWLEDGE_ORGANIZATION_RESTRICT)) {
+        if (relType === RELATION_GRANTED_TO && !isUserHasCapability(user, KNOWLEDGE_ORGANIZATION_RESTRICT)) {
```

Comment:
Can we refactor `isUserHasCapability` to `doesUserHasCapability` ?

Reply:
yes I agree that's what it seems to me too